example/workflows.yml

```diff
@@ -356,16 +356,35 @@ IngestGranule:
             - States.ALL
           IntervalSeconds: 2
           MaxAttempts: 3
+      Next: MoveGranuleStep
+    MoveGranuleStep:
+      CumulusConfig:
+        bucket: '{$.meta.buckets.internal}'
+        buckets: '{$.meta.buckets}'
+        granuleIdExtraction: '{$.meta.collection.granuleIdExtraction}'
+        input_granules: '{$.meta.input_granules}'
+        collection: '{$.meta.collection}'
+        cumulus_message:
+          input: '{$.payload}'
+          outputs:
```

Comment:
If `outputs` is not specified, the output will go payload.  It seems redundant here.